From b87e312cba8cd1ea8b08fd33fba86cc19fcafb1b Mon Sep 17 00:00:00 2001
Message-Id: <b87e312cba8cd1ea8b08fd33fba86cc19fcafb1b.1367758397.git.jlec@gentoo.org>
From: Justin Lecher <jlec@gentoo.org>
Date: Sun, 5 May 2013 14:51:01 +0200
Subject: [PATCH] Detect tgetent in libtinfo

For linux when build ncurses with --with-termlib you get libtinfo.so by
default.
This patch extends detection of tgetent to libtinfo

Signed-off-by: Justin Lecher <jlec@gentoo.org>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index ebcb993..5bcc545 100644
--- a/configure.in
+++ b/configure.in
@@ -24,6 +24,7 @@ AC_CHECK_HEADERS(socket.h sys/socket.h sys/xti.h arpa/nameser_compat.h)
 # We have to trust the linker not to mess things up... (It should not
 # pull in anything if we don't refer to anything in the lib). 
 AC_CHECK_LIB(termcap, tgetent)
+AC_CHECK_LIB(tinfo, tgetent)
 
 AC_CHECK_FUNC(initscr, , 
   AC_CHECK_LIB(ncurses, initscr, , 
-- 
1.8.2.1

